packages/flutter/pubspec.yaml

```diff
@@ -28,9 +28,6 @@ dependencies:
   sembast: ^3.2.0
   sembast_web: ^2.0.1+1
   path: ^1.8.1 # required for transitive use only
-
-dependency_overrides:
-  path: ^1.8.2 # required for transitive use only
   platform: ^3.1.0
```

Comment:
should platform also be removed?

Reply:
não. foi removido de dependency_overrides e deixado apenas em dependency

Reply:
not.  was removed from dependency_overrides and left only in dependency

Reply:
got it